<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Underground</class>
 <widget class="QWidget" name="Underground">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Underground</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_45">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_42">
      <item row="2" column="1">
       <widget class="ComboBox" name="comboBoxLocation"/>
      </item>
      <item row="3" column="1">
       <widget class="CheckList" name="checkListPokemon"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLocation">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxStoryFlag">
        <item>
         <property name="text">
          <string>Underground Unlocked</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Strength Obtained</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Defog Obtained</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Waterfall Obtained</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>National Dex</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelStoryFlag">
        <property name="text">
         <string>Story Flag</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPokemon">
        <property name="text">
         <string>Pok√©mon</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxDiglett">
        <property name="text">
         <string>Diglett Bonus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLevelFlag">
        <property name="text">
         <string>Level Flag</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLevelFlag">
        <item>
         <property name="text">
          <string>0/1 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 Badges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>National Dex</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_44">
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Brilliant Diamond</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="TableView" name="tableView"/>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxFilters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_41">
      <item row="0" column="0" rowspan="2">
       <widget class="Filter" name="filter" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRNGInfo">
     <property name="title">
      <string>RNG Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_43">
      <item row="5" column="3">
       <widget class="TextBox" name="textBoxInitialAdvances">
        <property name="text">
         <string notr="true">0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelLead">
        <property name="text">
         <string>Lead</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="ComboMenu" name="comboMenuLead"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelSeed1">
        <property name="text">
         <string>Seed 1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="TextBox" name="textBoxSeed1"/>
      </item>
      <item row="6" column="3">
       <widget class="TextBox" name="textBoxMaxAdvances">
        <property name="text">
         <string notr="true">100000</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="3">
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelSeed0">
        <property name="text">
         <string>Seed 0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelInitialAdvances">
        <property name="text">
         <string>Initial Advances</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="TextBox" name="textBoxSeed0"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelMaxAdvances">
        <property name="text">
         <string>Max Advances</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="labelDelay">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="TextBox" name="textBoxDelay"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboMenu</class>
   <extends>QToolButton</extends>
   <header>Form/Controls/ComboMenu.hpp</header>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>comboMenuLead</tabstop>
  <tabstop>textBoxSeed0</tabstop>
  <tabstop>textBoxSeed1</tabstop>
  <tabstop>textBoxInitialAdvances</tabstop>
  <tabstop>textBoxMaxAdvances</tabstop>
  <tabstop>textBoxDelay</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxStoryFlag</tabstop>
  <tabstop>comboBoxLevelFlag</tabstop>
  <tabstop>comboBoxLocation</tabstop>
  <tabstop>checkListPokemon</tabstop>
  <tabstop>checkBoxDiglett</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
